From 60bbff88eaa811b34cd5eed79a3264159380f547 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Thu, 20 Jul 2017 12:08:25 +0100
Subject: [PATCH 10/10] context: Stop using GCC 7 -fprofile-update=atomic flag

Solus has now moved back to GCC 6.4 so this flag is no longer recognised

Signed-off-by: Ikey Doherty <ikey@solus-project.com>
---
 ypkg2/ypkgcontext.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/ypkg2/ypkgcontext.py b/ypkg2/ypkgcontext.py
index 3da5b50..cceefd7 100644
--- a/ypkg2/ypkgcontext.py
+++ b/ypkg2/ypkgcontext.py
@@ -32,8 +32,7 @@ BIND_NOW_FLAGS = ["-Wl,-z,now", "-Wl,-z -Wl,relro", "-Wl,-z -Wl,now"]
 SIZE_FLAGS = "-Os -ffunction-sections"
 
 # GCC PGO flags
-PGO_GEN_FLAGS = "-fprofile-generate -fprofile-dir=\"{}\" " \
-                "-fprofile-update=atomic"
+PGO_GEN_FLAGS = "-fprofile-generate -fprofile-dir=\"{}\" "
 PGO_USE_FLAGS = "-fprofile-use -fprofile-dir=\"{}\" -fprofile-correction"
 
 # Clang can handle parameters to the args unlike GCC
-- 
2.13.3

